README.md

```diff
@@ -99,20 +99,46 @@ variable. After that you can run the following lines of code:
 All packages can be installed via `install.packages`
 
 ```
-pkgs <- c("gdxrrw",
-          "ggplot2",
-          "curl",
+pkgs <- c("colorspace",
+          "crayon",
+          "data.table",
+          "devtools",
+          "dplyr",
+          "edgeTrpLib",
+          "flexdashboard",
           "gdx",
-          "magclass",
-          "madrat",
-          "mip",
-          "lucode2",
+          "gdxdt",
+          "gdxrrw",
+          "ggplot2",
           "gms",
-          "remind2",
-          "lusweave",
-          "luscale",
           "goxygen",
+          "gridExtra",
+          "gtools",
+          "htmltools",
+          "knitr",
+          "lazyeval",
+          "lucode2",
           "luplot",
+          "luscale",
+          "lusweave",
+          "magclass",
+          "magpie4",
+          "mip",
+          "mrremind",
+          "mrvalidation",
+          "optparse",
+          "plotly",
+          "quitte",
+          "remind",
```

Comment:
Do we still depend on remind as well as remind?

Reply:
Nope, #785

Reply:
Did you already test the suggestion from the issue? Can I just add your suggestion to this PR?

Reply:
I have not tested it, but I think it will work.

Reply:
Could you test it? I think this is much quicker than if I try to find out how to call the function now

Reply:
I tested it, looks fine to me.